lib/CabalToDhall.hs

```diff
@@ -122,14 +134,18 @@ cabalToDhall source =
 
     Cabal.ParseOk _warnings genericPackageDescription -> do
       let
+        body = Dhall.TypeCheck.absurd <$>
+          Dhall.embed
+            genericPackageDescriptionToDhall
+            genericPackageDescription
         dhall =
-          Expr.Let "prelude" Nothing ( Expr.Embed preludeLocation )
-            $ Expr.Let "types" Nothing ( Expr.Embed typesLocation )
-            $ ( Dhall.TypeCheck.absurd <$>
-                Dhall.embed
-                  genericPackageDescriptionToDhall
-                  genericPackageDescription
-              )
+          case importType of
+            ImportFromGitHub ->
+              Expr.Let "prelude" Nothing ( Expr.Embed preludeLocation )
+                $ Expr.Let "types" Nothing ( Expr.Embed typesLocation )
+                $ body
+            NoImports ->
+              body
```

Comment:
I don't quite understand why I'd want to do this - it just produces a Dhall file that won't type check (due to loads of missing prelude references).

Reply:
That's for test output stability, so that each version bump doesn't mean that all the cabal-to-dhall test outputs need bumping, even if nothing substantial changed.

Reply:
Yea, unfortunately it's part of `exposed-modules` though so I don't feel great about exposing an API that produces something fairly "obviously wrong". What do you think about supplying explict Dhall imports to the prelude & types? Then the exe can supply GitHub fetches, and the tests can supply relative file paths.

Reply:
Oh, hmm. What about an internal library, so it's not part of the public API?

Reply:
Yea, I think that would also work. `CabalToDhall` would have the same type as it does now, but it'd call into the internal-library, that the `exe` would also call directly into? A minor shame to not have the exe actually consume the public API, but I could live with that.